File: ws/codena/u3/ws29SingletonCode/Python/TaxCalculator/tax_calculator.py
from tax import Tax

iva = Tax.getInstance()

anotherTax = Tax.getInstance()

percentage = 0.15
price = 1000

iva.updateTaxPercentage(percentage)

calculatedPrice = iva.salesTotal(price)

print("The percentage is:", iva.getPercentage())
print("The price is:", f"{calculatedPrice}$")

File: ws/codena/u3/ws29SingletonCode/Javascript/TaxCalculator/taxCalculator.js
import Tax from "./Tax.js";

const iva = Tax.getInstance();

const anotherTax = Tax.getInstance();

const percentage = 0.15;
const price = 1000;

iva.updateTaxPercentage(percentage);

const calculatedPrice = iva.salesTotal(price);

console.log("The percentage is:", iva.getPercentage());
console.log("The price is:", calculatedPrice + "$");

